### PR TITLE
core function 0607

### PR DESCRIPTION
**Pull request by ethanlcz**

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
start build

**Comment by skylarCai** [comment]
手动触发！
感谢 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : 2cfff34c7b2169a94c85aa0879e08d81812b76ef

**Comment by skylarCai** [comment]
auto build success,number:208

**Comment by skylarCai** [comment]
autotest fail,number:1559

**Review by wushengtws — APPROVED** [review]

File: haas_lib_bundles/python/docs/core_function/HaaS_Python_app_ota.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+### 创建工程
+通过HaaS Studio基于helloworld创建名为“HaaS_OTA”工程。
+### 设备联网
+HaaS在线更新需要开发板接入互联网，不同的平台的接入方式请参考如下：
+#### ESP32系列（NodeMCU32S、ESP32乐鑫、pyWiFi-ESP32、M5StackCore2）
+复制如下代码到"HaaS_OTA"工程中的main.py中，并修改源码中的wifiSsid & wifiPassword为自己的路由器热点信息，再通过“本地更新”方式将示例代码推送到硬件中执行，等待设备联网成功（打印“Wi-Fi connected”）。
+```python
+import network                   # Wi-Fi功能所在库
+import utime                     # 延时API所在组件
+
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/core_function/HaaS_Python_app_ota.md:43]: LGFM

File: haas_lib_bundles/python/docs/core_function/HaaS_Python_app_ota.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    utime.sleep_ms(500)
+print('Wi-Fi connected', sta_if.ifconfig())
+```
+#### HaaS EDU K1和HaaS200
+复制如下代码到"HaaS_OTA"工程中的main.py中，并修改源码中的wifiSsid & wifiPassword为自己的路由器热点信息，再通过“本地更新”方式将示例代码推送到硬件中执行，等待设备联网成功（打印“Wi-Fi connected”）。
+```python
+# -*- coding: UTF-8 -*-
+import netmgr as nm              # Wi-Fi功能所在库
+import utime                     # 延时API所在组件                # ir人体红外传感器类
+
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/core_function/HaaS_Python_app_ota.md:64]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**